src/MIDebugEngine/MIDebugEngine.csproj

```diff
@@ -141,6 +141,7 @@
     <Compile Include="Engine.Impl\EngineCallback.cs" />
     <Compile Include="Engine.Impl\EngineUtils.cs" />
     <Compile Include="Engine.Impl\ExceptionManager.cs" />
+    <Compile Include="AD7.Impl\HashAlgorithmId.cs" />
```

Comment:
This does not seem to be placed properly by VS? Whenever a change is made to the csproj file, this will be moved.